editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -0,0 +1,80 @@
+import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
+import { useFormik } from "formik";
+import React from "react";
+import InputGroup from "ui/editor/InputGroup";
+import ModalSection from "ui/editor/ModalSection";
+import ModalSectionContent from "ui/editor/ModalSectionContent";
+import RichTextInput from "ui/editor/RichTextInput";
+import Input from "ui/shared/Input";
+import InputRow from "ui/shared/InputRow";
+
+import { EditorProps, ICONS, InternalNotes, MoreInformation } from "../ui";
+import { MapAndLabel, parseContent } from "./model";
+
+type Props = EditorProps<TYPES.MapAndLabel, MapAndLabel>;
+
+export default MapAndLabelComponent;
+
+function MapAndLabelComponent(props: Props) {
+  const formik = useFormik({
+    initialValues: parseContent(props.node?.data),
+    onSubmit: (newValues) => {
+      props.handleSubmit?.({
+        type: TYPES.MapAndLabel,
+        data: newValues,
+      });
+    },
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit} id="modal">
+      <ModalSection>
+        <ModalSectionContent
+          title="Map and label"
+          Icon={ICONS[TYPES.MapAndLabel]}
+        >
+          <InputGroup>
+            <InputRow>
+              <Input
+                format="large"
+                name="title"
+                placeholder={"Title"}
+                value={formik.values.title}
+                onChange={formik.handleChange}
+              />
```

Comment:
nit: one final thing I noticed while manually testing in the Editor - I can currently create a new component without entering any input, which doesn't match the types you've defined in `model.ts` (eg I'd expect `title` & `fn` to be required and throw an error if no value!)

Simply adding the `required` prop on the `Input` should do the trick! (Will just throw a native browser error if empty then, not full Gov UK styled error - but that's fine for the Editor for now)

Reply:
Great! I'll update before merging